h2o-core/src/main/java/water/H2O.java

```diff
@@ -2173,7 +2172,19 @@ public static boolean checkUnsupportedJava() {
       return true;
     }
     return false;
-  } 
+  }
+
+  /**
+   * Any system property starting with `ai.h2o.` and containing any more `.` does not match
+   * this pattern and is therefore ignored. This is mostly to prevent system properties
+   * serving as configuration for H2O's dependencies (e.g. `ai.ai.h2o.org.eclipse.jetty.LEVEL` ).
```

Comment:
`ai.ai.h2o.org.eclipse.jetty.LEVEL` -> `ai.h2o.org.eclipse.jetty.LEVEL` or `sys.ai.h2o.org.eclipse.jetty.LEVEL` like in the test?

Reply:
good catch! thank you

It should be `ai.h2o.org.eclipse.jetty.LEVEL`